### PR TITLE
Feature/issue 114

### LINKED ISSUE #114: [Logic] Meeple placing
I want a function that connects player to chosen project using a meeple.

- [ ] after placing a tile a meeple connected to user is available
- [ ] a meeple can be attached to any of tile's projects that are not taken by other players

### PR DESCRIPTION
**Pull request by Urszuja**

### CODE REVIEW

**Comment by Urszuja** [comment]
image

should we also chceck current game phase?

File: frontend/src/components/GameBoard/GameBoard.functions.ts [hunk 1]
```diff
@@ -204,3 +203,26 @@ export const extendBoard = (row: number, column: number) => {
...
+    const projectOfLocation = availableProjects.find((p: Project) => p.type === location);
+    if (projectOfLocation) {
+      projectOfLocation.meeples.push(meeple);
+    }
+  } else if (availableProjects) {
+    console.log('no meeples available');
+    //end of turn
+  } else {
+    console.log('no available projects');
+  }
```
Comment (Nefariusek) [inline frontend/src/components/GameBoard/GameBoard.functions.ts]: It is hard to read and debug. Also, we do not expect `no available projects` (no dropdown shown if it happens) 
We can dispatch invalid move event here
```js
if (!availableProjects) {
  //dispatch and return 
}

if (!meeples) {
  //dispatch and return
}


// and at this point, when all invalid scenarios are checked, we can simply do the placement action
```

File: frontend/src/stores/PlayersStore.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    this.players.push(newPlayer);
+  }
+
+  changeOrderOfPlayers(i: number) {
+    [this.players[i - 1], this.players[i]] = [this.players[i], this.players[i - 1]];
+  }
+  getCurrentPlayer() {
+    return this.players.find(
+      (_player, index) => index === (this.rootStore.gameStore.turnNumber - 1) % this.players.length,
+    );
```
Comment (Nefariusek) [inline frontend/src/stores/PlayersStore.ts:30]: wouldn't it be easier to use just a this.player[modulo of turnnumber]? You are not using _player anyway so .find seems to be an overkill

**Review by Nefariusek — CHANGES_REQUESTED** [review]

File: frontend/src/stores/PlayersStore.ts (hunk 2 above)
Comment (Urszuja) [reply frontend/src/stores/PlayersStore.ts:30]: k

**Review by Nefariusek — APPROVED** [review]

**Pull request merged**